src/task1.js

```diff
@@ -0,0 +1,18 @@
+import readline from "readline";
+
+const task1 = () => {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    terminal: false,
+  });
+
+  rl.on("line", (line) => {
```

Comment:
Missed this in the previous PR.
Consider extracting handlers into a separate named functions and splitting the call chain into separate `rl.on(...)` pieces. That does help emphasise the event-driven nature of code and in makes it easier and cleaner to extend with other events. Moreover, error handling / closing is usually placed at the bottom of the file / logic - think of if-else statements where "else" part is the last one.

Reply:
Cool, how is this Dmitry?

```
  rl.on("line", reverseString);
  rl.on("error", handleError);
  rl.on("close", handleClose);
```